resolwe_bio/processes/variant_calling/gatk_select_variants.py

```diff
@@ -161,3 +184,166 @@ def run(self, inputs, outputs):
         outputs.tbi = selected_variants_index
         outputs.species = species
         outputs.build = build
+
+
+class GatkSelectVariantsSingleSample(Process):
+    """Select a subset of variants based on various criteria using GATK SelectVariants.
+
+    This tool works with single-sample VCF file as an input.
+    """
+
+    slug = "gatk-select-variants-single"
+    process_type = "data:variants:vcf:selectvariants:single"
+    name = "GATK SelectVariants (single-sample)"
+    version = "1.0.0"
+    entity = {
+        "type": "sample",
+    }
+    category = "GATK"
+    scheduling_class = SchedulingClass.BATCH
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/dnaseq:6.3.1"}
+        },
+        "resources": {
+            "cores": 2,
+            "memory": 16384,
+            "storage": 200,
+        },
+    }
+
+    data_name = "Selected variants"
```

Comment:
Since the process is now only for a single sample, maybe we could use something like `data_name = "Selected variants ({{ vcf|sample_name|default("?") }})"`.